python/tvm/relay/backend/vmobj.py

```diff
@@ -133,24 +107,3 @@ def tuple_object(fields):
     for f in fields:
         assert isinstance(f, Object)
     return _vmobj.Tuple(*fields)
```

Comment:
I feel that we can just use Datatype with tag=0 instead of using `_vmobj.Tuple`. it seems less confusing?

Reply:
I think it would still be helpful to have an auxiliary function that constructs the tuple

Reply:
sure